pkg/policy/repository_deny_test.go

```diff
@@ -817,8 +817,8 @@ func TestL3DependentL4EgressDenyFromRequires(t *testing.T) {
 			},
 			RuleOrigin: map[CachedSelector]labels.LabelArrayList{expectedCachedSelector: {nil}},
 		},
-	}
-	if !assert.EqualValues(t, expectedDenyPolicy, policyDeny) {
+	})
+	if !assert.True(t, expectedDenyPolicy.Equals(t, policyDeny), policyDeny.Diff(t, expectedDenyPolicy)) {
```

Comment:
Does not have a functional difference, but would be nice to have `policyDeny. Equals(t, expectedDenyPolicy)` here to have visual balance with the `Diff` call after.